recipes/pysirius/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.9" %}
```

Comment:
If this is version `0.9`, why is `pip` reporting:

```
Successfully installed PySirius-1.0.0
```

🤔

Reply:
@jaimergp This is caused due to swagger generating the code as a "package" with version 1.0.0. It does not matter that it does that, as the conda-forge package is still successfully build as `py-sirius-ms 0.9` (see i.e. line 437 in linux_64 "Run docker build"). When generating a new client API in the future, swagger will still call it PySirius-1.0.0, but our package may be `py-sirius-ms 1.3` at that point. 
TL;DR: we can ignore swaggers versioning and use our own

Reply:
but anyways, I will adjust it shortly so it matches

Reply:
Yea, this is not a blocker for the merge. You can amend it in-feedstock in the future.